Makefile

```diff
@@ -36,6 +37,13 @@ source-release: clean
 install:
 	$(PYTHON) setup.py install --root $(DESTDIR) $(COMPILE)
 
+pip:
+	$(PYTHON) -m pip --version || $(PYTHON) -c "import os; import sys; import urllib; f = urllib.urlretrieve('https://bootstrap.pypa.io/get-pip.py')[0]; os.system('%s %s' % (sys.executable, f))"
+
+requirements: pip
+	- pip install "pip>=6.0.1"
+	- pip install -r requirements.txt
```

Comment:
I'd suggest putting the `subprocess32` directly into `setup.py` as `install_requires` keyword and only if it's necessary (based on the suggested condition)

Reply:
ok.